activerecord/lib/active_record/associations.rb

```diff
@@ -504,7 +504,7 @@ def association_instance_set(name, association)
     #
     # == Customizing the query
     #
-    # \Associations are built from <tt>Relation</tt>s, and you can use the Relation syntax
+    # \Associations are built from <tt>Relations</tt>, and you can use the Relation syntax
```

Comment:
Better. `s` can't be part of the `<tt>` font here. @xw19 can you incorporate this change?